style.css

```diff
@@ -0,0 +1,31 @@
+body{
```

Comment:
Dear Feruz, If you change Body style from here, all elements will affect from this. Maybe we arrange padding and box-shadows at the end